exercises/02-module-class/color-generator-2.0.mjs

```diff
@@ -0,0 +1,25 @@
+class ColorsGenerator {
+
+    colors = ['red', 'green', 'blue'];
+    i = 0;
```

Comment:
Not a bug per se, but a bit more descriptive name would be nice 😅 For example `counter` or `index`.